### PR TITLE
fix: Classifier redaction from EBS

### PR DESCRIPTION
**Pull request by disq**
Also improve Request ID regex

EBS AccessDenied error still there.

### CODE REVIEW

**Review by erezrokah — APPROVED** [review]
Related to https://github.com/cloudquery/cq-provider-aws/issues/621. I'll look at EBS AccessDenied error as a part of https://github.com/cloudquery/cq-provider-aws/issues/621.

Thanks @disq for handling this 🎉

File: client/errors.go [hunk 1]
```diff
@@ -153,13 +153,14 @@ func isCodeThrottle(code string) bool {
 }
 
 var (
-	requestIdRegex = regexp.MustCompile(`\s(RequestID:|request id:)\s[A-Za-z0-9-]+`)
+	requestIdRegex = regexp.MustCompile(`\s(RequestID:|request id:|Request ID:)\s[A-Za-z0-9-]+`)
```
Comment (roneli) [inline client/errors.go:156]: nit: maybe make the regexes case insensitive?

**Review by roneli — APPROVED** [review]
LGTM

File: client/errors.go (hunk 1 above)
Comment (disq) [reply client/errors.go:156]: So `ReQuEsT iD` can also pass? It's too much I think 😂

Comment (disq) [reply client/errors.go]: but I fixed the regex for you

**Review by roneli — APPROVED** [review]

**Pull request merged**